File: integration-tests/hibernate-orm-panache/src/main/java/io/quarkus/it/panache/NamedQueryEntity.java
package io.quarkus.it.panache;

import jakarta.persistence.Entity;
import jakarta.persistence.NamedQuery;

@Entity
@NamedQuery(name = "NamedQueryEntity.getAll", query = "from NamedQueryEntity")
public class NamedQueryEntity extends NamedQueryMappedSuperClass {
    public String test;
}

File: integration-tests/hibernate-reactive-panache-kotlin/src/main/kotlin/io/quarkus/it/panache/reactive/kotlin/NamedQueryEntity.kt
package io.quarkus.it.panache.reactive.kotlin

import io.quarkus.hibernate.reactive.panache.kotlin.PanacheCompanion
import jakarta.persistence.Entity
import jakarta.persistence.NamedQuery

@Entity
@NamedQuery(name = "NamedQueryEntity.getAll", query = "from NamedQueryEntity")
class NamedQueryEntity : NamedQueryMappedSuperClass() {
    lateinit var test: String

    companion object : PanacheCompanion<NamedQueryEntity>
}
